test/ZendeskApi.Client.Tests/Resources/TicketFormsResourceTests.cs

```diff
@@ -24,101 +22,134 @@ public TicketFormsResourceTests()
         }
 
         [Fact]
-        public async Task ShouldListAllTicketForms()
+        public async Task GetAllAsync_WhenCalled_ShouldGetAll()
         {
-            var obj1 = await _resource.CreateAsync(new TicketForm {
-                Name = "Superman1"
-            });
+            var results = await _resource.GetAllAsync();
 
-            var obj2 = await _resource.CreateAsync(new TicketForm
-            {
-                Name = "Superman2"
-            });
+            Assert.Equal(100, results.Count);
```

Comment:
I seem to notice this in every PR. Could we posibily split it out, and give it the Assertions (e.g `Assert.Equal(i, item.Id);`) as action parameters?